### PR TITLE
Support versioning of ramda-repl js file

### PR DESCRIPTION
**Pull request by zehua**
We started to use `rawgit` since https://github.com/ramda/ramda.github.io/pull/136. From the comment in https://github.com/ramda/repl/pull/50#issuecomment-331756404, [rawgit faq](https://github.com/rgrove/rawgit/blob/master/FAQ.md#how-long-does-the-cdn-cache-files-how-can-i-make-it-refresh-my-file) does not recommend using `master` as the tag as they do not clear cache. 

This PR attempts to use tag from `ramda-repl` instead of always using `master` to overcome this issue.

I also added a util script to generate the repl/index.html from handlerbars. Sorry that I could not figure out how to use the new pug based template introduced in https://github.com/ramda/ramda.github.io/pull/175. It also looks like the current repl/index.html has been manually modified in the past few commits. Feel free to update the pug template to support the same, if this PR is accepted.  @MattMS 

TODO:
 * [ ] merge https://github.com/ramda/repl/pull/51 and upload 1.1.0 of ramda/repl

@craigdallimore @buzzdecafe

### CODE REVIEW

**Comment by MattMS** [comment]
Thanks for the research on the rawgit versioning, I'm happy with the idea of adding the version number.

I don't really like reverting back to Handlebars, because then we lose the inheritance for common content of the site.

I'm also not a fan of including an unused dependency just to get the version number.
The simplest fix would be replacing `master` with `1.1.0` in [repl/index.pug](https://github.com/ramda/ramda.github.io/blob/master/repl/index.pug).
Otherwise you could add the `repl_tag` to [make_repl_index_html.js](https://github.com/ramda/ramda.github.io/blob/master/make_repl_index_html.js) and replace `master` with `#{repl_tag}`.
I prefer the former.

**Comment by zehua** [comment]
> I don't really like reverting back to Handlebars, because then we lose the inheritance for common content of the site.

I did give the pug approach a try. But there's some styling issue with the width of the repl.

image

In addition, it looks like the current production site is still using the old html and styles. I thought it's safer to just update based on the older version and wait for someone more familiar with the pug approach to proper test it and release it.

> I'm also not a fan of including an unused dependency just to get the version number.
> The simplest fix would be replacing master with 1.1.0 in repl/index.pug.
> Otherwise you could add the repl_tag to make_repl_index_html.js and replace master with #{repl_tag}.
> I prefer the former.

In this sense, we are also including `ramda` itself as an unused dependency, since we are really only using its version. I like explicit dependency. But this is a minor issue. I am ok with including `1.1.0` directly in the pug template.

In short, do you prefer me to directly update the `repl/index.html` with the new `1.1.0` tag in this PR and you taking another look at fixing the pug and less styling issue and launch the PUG approach properly in a separate PR? @MattMS   I could take a stab at fixing the styling issue (it's probably [this change](https://github.com/ramda/ramda.github.io/pull/185/files#diff-e73ad0a6a3cca0e9ce1279bf47761a0eR45)). But it's up to the site maintainer to decide when to release this to production site.

**Comment by zehua** [comment]
I've updated the PR to just use the `1.1.0` value directly in the html.

I updated both `repl/index.html` for immediate release now, and also `repl/index.pug` for future release of the PUG based version.

The styling issue with PUG repl is addressed in a separate PR: https://github.com/ramda/ramda.github.io/pull/191

Let me know whether this works for you @MattMS

**Comment by zehua** [comment]
ping reviewers

**Comment by MattMS** [comment]
Thanks @zehua, this all looks fine to me.

I'll check out your other PRs next.

**Comment by MattMS** [comment]
REPL with that version was not deployed.
I've reverted this until https://github.com/ramda/repl/pull/51 lands.
